include/logger/loggerConfig.h

```diff
@@ -318,6 +318,9 @@ typedef struct _OBD2Config {
 typedef struct _CANConfig {
     unsigned char enabled;
     int baud[CONFIG_CAN_CHANNELS];
+#if CAN_SW_TERMINATION == true
```

Comment:
Does this need this `#ifdef` or can we just leave the termination Boolean array there in even in platforms that don't support it?  Train of thought here is to minimize `#ifdefs` in code where reasonable.